crates/re_types/definitions/rerun/components/marker_shape.fbs

```diff
@@ -7,28 +7,18 @@ include "rerun/attributes.fbs";
 
 namespace rerun.components;
 
-// TODO(#3384)
-/*
-enum MarkerShape: byte {
-    Circle = 1,
-    Diamond = 2,
-    Square = 3,
-    Cross = 4,
-    Plus = 5,
-    Up = 6,
-    Down = 7,
-    Left = 8,
-    Right = 9,
-    Asterisk = 10,
-}
-*/
-
 /// Shape of a marker.
-struct MarkerShape (
-    "attr.python.aliases": "int, str",
-    "attr.python.array_aliases": "int, str",
-    "attr.rust.derive": "PartialEq, Eq, PartialOrd, Copy",
-    "attr.rust.repr": "transparent"
+enum MarkerShape: byte (
+    "attr.docs.unreleased"
```

Comment:
why unreleased?

Reply:
The generated doclink changes from https://docs.rs/re_types/latest/re_types/components/struct.MarkerShape.html to https://docs.rs/re_types/latest/re_types/components/enum.MarkerShape.html which doesn't exist (yet)